nolibc/include/signal.h

```diff
@@ -4,14 +4,24 @@
 typedef int jmp_buf;
 int setjmp(jmp_buf);
 void (*signal(int sig, void (*func)(int)))(int);
-#define SIG_DFL NULL
-#define SIG_IGN NULL
-#define SIG_ERR NULL
+#define SIG_DFL 0
+#define SIG_IGN 0
+#define SIG_ERR 0
+#define SIG_BLOCK 0
+#define SIG_SETMASK 0
 /*
  * The following definitions are not required by the OCaml runtime, but are
  * needed to build the freestanding version of GMP used by Mirage.
+ * For OCaml 5.0.0, it's not totally true. SIG_{BLOCK,SETMASK,IGN,DFL) are
+ * needed by the OCaml runtime.
```

Comment:
```suggestion
 * The following definitions are required to build the freestanding version of GMP used by Mirage.
OCaml 5.0.0 also required the SIG_{BLOCK,SETMASK,IGN,DFL) definition.
```
We rephrased part of the comment that was outdated by OCaml 5.0.0 support.